frontend/src/validation/index.js

```diff
@@ -62,9 +103,42 @@ const visitSchema = Yup.object().shape({
     .integer(),
 })
 
-const validateVisitForm = data => {
+// eslint-disable-next-line no-unused-vars
+const participantSchema = Yup.object().shape({
+  first_name: Yup.string()
+    .required()
+    .max(50),
+  last_name: Yup.string()
+    .required()
+    .max(100),
+  date_of_birth: Yup.string().required(),
```

Comment:
Id like to make these dates valid for a reasonable period, between 1900 and today. i messed around with it and we have to turn the date string back into a date object, or extend the schema and handle it ourselves. 

idea is to add a new case to handleParticipant update fn, that just dumps the date string `yyyy-MM-dd` into new Date(),  then update this to yup.date() with min and max.

i tested it and it submitted successfully to the back end, though it seems a little silly? /shrug